### PR TITLE
Refactor formatTime into util class

### PR DESCRIPTION
**Pull request by rejas**
While looking at https://github.com/MichMich/MagicMirror/pull/3070 I noticed that the weather and clock module do some formatTime stuff, so why not use a common function for that?

### CODE REVIEW

**Review by KristjanESPERANTO — APPROVED** [review]
Nice! I haven't tested it, but it looks good to me.

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Special thanks to @khassel, @rejas and @sdetweil for taking over most (if not al
 - Update Eslint config, add new rule and handle issue
 - Convert lots of callbacks to async/await
 - Revise require imports (#3071 and #3072)
+- Refactor formatTime into common util function for default modules
```
Comment (KristjanESPERANTO) [inline CHANGELOG.md:65]: That's for 2.23.

Comment (rejas) [reply CHANGELOG.md:65]: Good catch :-)

**Review by khassel — APPROVED** [review]

**Comment by khassel** [comment]
> Running tests with times/dates is a PITA ... Had to add an action to the github-actions for setting the timezone, otherwise the tests would fail here (while being fine locally)...

hopefully this helps ...

**Comment by khassel** [comment]
now I understood that only the new tests need the timezone (because these tests are now failing on my side). Is there no way to get these test running without needing a specific timezone? Or move them to the `electron` section where we can handle such things already?

**Comment by khassel** [comment]
found a jest solution, will provide a new PR ...

**Pull request merged**